pkg/pillar/types/newlogtypes.go

```diff
@@ -65,6 +70,7 @@ type NewlogMetrics struct {
 	NumKmessages          uint64            // total input kmessages
 	NumSyslogMessages     uint64            // total input syslog message
 	DevTop10InputBytesPCT map[string]uint32 // top 10 sources device log input in percentage
+	TotalSizeLogs         uint64            // total size of logs on device
```

Comment:
we have like 'NumSkipUploadAppFile', maybe we can also record the number of uploaded files.

Reply:
I think it's already recorded in `NumGZipFilesSent`